python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -365,6 +373,33 @@ def run(self, science, matchedTemplate, difference,
                                    positiveFootprints=positives,
                                    negativeFootprints=negatives)
 
+    def _prepareInputs(self, difference):
+        """Ensure that we start with an empty detection and deblended mask.
+
+        Parameters
+        ----------
+        difference : `lsst.afw.image.ExposureF`
+            The difference image that will be used for detecting diaSources.
+            The mask plane will be modified in place.
+
+        Returns
+        -------
+        sigma : `float`
+            The width of the PSF of the difference image.
```

Comment:
As noted elsewhere, I think you can remove this return if you just log that info message at the end of this method.